Description: update apparmor profile for userns permission and new abi
Author: Alex Murray <alex.murray@canonical.com>
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/apparmor/+bug/2035315
--- a/config/apparmor/usr.bin.lxc-start
+++ b/config/apparmor/usr.bin.lxc-start
@@ -1,3 +1,4 @@
+abi <abi/4.0>,
 #include <tunables/global>
 
 /usr/bin/lxc-start flags=(attach_disconnected) {
--- a/config/apparmor/abstractions/start-container.in
+++ b/config/apparmor/abstractions/start-container.in
@@ -1,3 +1,4 @@
+  userns,
   network,
   capability,
   file,
